client/layout/guided-tours/design-showcase-welcome-tour.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+	negate as not,
+} from 'lodash';
+import { isDesktop } from 'lib/viewport';
+
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Continue,
+} from 'layout/guided-tours/config-elements';
+import {
+	isAbTestInVariant,
+	inSection,
+	isNewUser,
+	isEnabled,
+	selectedSiteIsCustomizable,
+	userHasInteractedWithComponent,
+} from 'state/ui/guided-tours/contexts';
+
+const anyThemeMoreButtonClicked = state => {
+	return userHasInteractedWithComponent( 'ThemeMoreButton' )( state );
+};
+
+export const DesignShowcaseWelcomeTour = makeTour(
+	<Tour
+		name="designShowcaseWelcome"
+		version="20161123"
+		path="/design"
+		when={ and(
+			isNewUser,
+			isEnabled( 'guided-tours/design-showcase-welcome' ),
+			isDesktop,
+			selectedSiteIsCustomizable,
+			not( inSection( 'customize' ) ),
+			isAbTestInVariant( 'designShowcaseWelcomeTour', 'enabled' ),
+			) }
+		>
```

Comment:
For consistency with the `Step`s below, let's indent both lines here one increment to the left, `) }` and `>`.